packages/flutter/lib/src/material/dropdown.dart

```diff
@@ -760,6 +838,12 @@ class DropdownButton<T> extends StatefulWidget {
   /// surrounding container.
   final bool isExpanded;
 
+  /// If null, then the menu item heights will vary according to each menu item's
+  /// intrinsic height.
+  ///
```

Comment:
Is it worth mentioning here that if the menu is too tall for the screen and `itemHeights` is set to `null`, the scroll offset will not be properly set and always be at zero?

Reply:
Yes, sadly yes. Updated the doc to explain this.